[enterprise-4.13] DOCS OBSDOCS-658 -Logging 5.6.14 Release Notes (#69237)
openshift/openshift-docs: modules/logging-release-notes-5-6-14.adoc

// Module included in the following assemblies:
// logging-5-6-release-notes.adoc
:_content-type: REFERENCE
[id="logging-release-notes-5-6-14_{context}"]
= Logging 5.6.14

> This release includes link:https://access.redhat.com/errata/RHSA-2023:7719[OpenShift Logging Bug Fix Release 5.6.14].

Author: Fixed this. The link is working fine now. Thank you!

[id="logging-release-notes-5-6-14-bug-fixes"]
== Bug fixes

> * Before this update, the default alias was missing from the initial index in each log output while creating index patterns. As a result, the Kibana users were unable to create index patterns using {es-op}. With this update, adding the missing aliases to {es-op} resolves the issue. As a result, the Kibana users can create index patterns that include the `{app,infra,audit}-000001` indices. (link:https://issues.redhat.com/browse/LOG-4807[LOG-4807])

Author: Fixed this. Thanks for pointing it out.

> * Before this update, the {loki-op} did not mount a custom CA Bundle to the ruler pods. As a result, it caused failing object storage access when evaluating alerting or recording rules. With this update, the {loki-op} mounts the custom CA bundle to all ruler pods. As a result, the ruler pods can download logs from object storage to evaluate alerting or recording rules. (link:https://issues.redhat.com/browse/LOG-4838[LOG-4838])

Reviewer: ```suggestion
* Before this update, the {loki-op} did not mount a custom CA bundle to the ruler pods. As a result, during the process to evaluate alerting or recording rules, object storage access failed. With this update, the {loki-op} mounts the custom CA bundle to all ruler pods. The ruler pods can download logs from object storage to evaluate alerting or recording rules. (link:https://issues.redhat.com/browse/LOG-4838[LOG-4838])
```
- I revised the second sentence to avoid a dangling modifier, per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#general-guidelines

Author: Fixed it. Thanks!

[id="logging-release-notes-5-6-14-CVEs"]

> == CVEs

Reviewer: I checked all of the links, and they all work. Nice job -- that was quite a list!

* link:https://access.redhat.com/security/cve/CVE-2007-4559[CVE-2007-4559]
* link:https://access.redhat.com/security/cve/CVE-2021-43975[CVE-2021-43975]

> * link:https://access.redhat.com/security/cve/CVE-2022-3594[CVE-022-3594]
> * link:https://access.redhat.com/security/cve/CVE-2022-3640[CVE-2022-3640]
> * link:https://access.redhat.com/security/cve/CVE-2022-4744[CVE-022-4744]
> * link:https://access.redhat.com/security/cve/CVE-2022-28388[CVE-2022-28388]
> * link:https://access.redhat.com/security/cve/CVE-2022-38457[CVE-022-38457]

Reviewer 2: ```suggestion
* link:https://access.redhat.com/security/cve/CVE-2022-3594[CVE-2022-3594]
* link:https://access.redhat.com/security/cve/CVE-2022-3640[CVE-2022-3640]
* link:https://access.redhat.com/security/cve/CVE-2022-4744[CVE-2022-4744]
* link:https://access.redhat.com/security/cve/CVE-2022-28388[CVE-2022-28388]
* link:https://access.redhat.com/security/cve/CVE-2022-38457[CVE-2022-38457]
```

Author: Made this change! Thanks.
